test/common/integration/rtds_integration_test.cc

```diff
@@ -85,6 +85,7 @@ TEST_P(RtdsIntegrationTest, RtdsReload) {
   EXPECT_EQ("yar", getRuntimeKey("bar"));
   EXPECT_EQ("", getRuntimeKey("baz"));
 
+  const uint64_t initial_load_success = getCounterValue("runtime.load_success");
```

Comment:
can we have a race where this may or may not already be set? i.e. should we wait for it to get incremented here, and just always hav the below one be 1?

Reply:
Probably no race there since the PostInit would've finished executing by then, but I think hard coding 1 is clearer so I changed it below per your suggestion. Thanks!